hardware/raspberrypi/usb/README.md

```diff
@@ -2,13 +2,15 @@
 
 ## Page Contents
 
-- [Overview for devices prior to the Pi 4](#overview)  
-- [Overview for the Pi 4](#overview_pi4)   
-- [Supported devices](#support)
-- [General limitations (not Pi 4)](#genlimits)
-- [Port power limits](#powerlimits)
-- [Known issues (not Pi 4)](#knownissues)
+- [Overview for devices prior to Pi 4](#overview-for-devices-prior-to-pi-4)
```

Comment:
The anchors in this section need to be reverted to what they used to be. `#overview-for-devices-prior-to-pi-4` will only work on github, not raspberrypi.org/documentation

Reply:
I still have no idea where they came from.